pkgs/top-level/all-packages.nix

```diff
@@ -33015,7 +33015,7 @@ with pkgs;
   };
   inherit (callPackage ../applications/networking/cluster/k3s {
     buildGoModule = buildGo120Module;
-  }) k3s_1_26 k3s_1_27 k3s_1_28;
+  }) k3s_1_26 k3s_1_27 k3s_1_28 k3s_1_29;
```

Comment:
k3s 1.29 is built with Go `1.21`.
https://github.com/k3s-io/k3s/blob/v1.29.0%2Bk3s1/go.mod#L3

Reply:
Well, fun. It built fine with 1.20 as well it seems. Will draft it and take a look this week to address it if no one else jumps on it.